www/less/widenImages.less

```diff
@@ -4,31 +4,9 @@
     display: block;
     margin-left: auto;
     margin-right: auto;
-}
-
-@media (orientation: portrait) {
-    .wideImageOverride {
```

Comment:
See the `shouldWidenImage` JS function for the other things which are related to the image widening logic.